src/js/edu-benefits/5490/config/form.js

```diff
@@ -312,6 +314,9 @@ const formConfig = {
           },
           schema: {
             type: 'object',
+            // If answered 'Yes' without entering information, it's the same as
+            //  answering 'No' as far as the back end is concerned.
+            required: ['view:applicantServed'],
```

Comment:
Is this going to be a problem for any reason? Is there a field in the schema I overlooked that this corresponds to? If not, should it really be required?